### ISSUE TITLE
Request: encode to hex

### ISSUE THREAD

**Issue by VerlindenEAM** [helpers/handlebars-helpers#406]
Would it be possible to get hex encoding helper?

We have some search related features using your Handlebars Helpers and it would be great to use the hex encoding in our URL mechanism.

For example:
"Analytics" would result in hex code "416e616c7974696373"
"Encode - Hex" would result in hex code "456e636f6465202d20486578"

var t = "Analytics"
var h = "";
for (let i=0; i<t.length; i++) {
h+= t.charCodeAt(i).toString(16);
}

console.log(t); // original text
console.log(h); // hex string

**Comment by BrinaD1** [comment]
Agreed, adding a vote to this. Definitely needed for use with the PnP Search web parts!
